source/common/runtime/runtime_features.cc

```diff
@@ -87,7 +133,8 @@ bool runtimeFeatureEnabled(absl::string_view feature) {
     IS_ENVOY_BUG(absl::StrCat("Unable to find runtime feature ", feature));
     return false;
   }
-  return absl::GetFlag(*flag);
+  // We validate in map creation that the flag is a boolean.
+  return flag->TryGet<bool>().value();
```

Comment:
GetFlag takes the absl flag and returns a commandlineflag.  Here we don't have the absl flag we only have the string name, so tryget allows us to get a coommandlineflag from that.

Reply:
AH! I see. That makes sense. I think I got confused because link 138 didn't change so I assumed that the type of flag didn't change. I think this is an example of how auto can actually make code a bit less readable, but it's a tradeoff, for sure. For bonus points consider changing auto to absl::CommandLineFlag on line 138. But totally at your discretion.